pkg/controller/controller_flow_test.go

```diff
@@ -59,6 +59,39 @@ func TestBasicFlowWithBasicAuth(t *testing.T) {
 	ct.AssertOSBBasicAuth(t, "user1", "newp2sswd")
 }
 
+// TestOriginatingIdentity tests whether the controller uses correct credentials when the secret changes
+func TestOriginatingIdentity(t *testing.T) {
```

Comment:
add info
`// CAUTION: the test cannot be executed in parallel because it changes global flag which can affect the behavior of other tests`